### ISSUE TITLE
Improve accessibility info provided on Event pages

### ISSUE THREAD

**Issue by jennpb** [wellcomecollection/wellcomecollection.org#4473]
## Who it's for
- Venue visitors who need to know whether they can/should attend an event

## Problem to be solved
We know from research that while most people with access needs (who are familiar with museums) will look for an Accessibility page on a website, it makes things much easier when we show access info in context of an exhibition or event.

There is currently a mismatch between what we show on an Event page and what we show on an Exhibition page. 

**Event**
![Event_needtoknow](https://user-images.githubusercontent.com/1446139/57845112-3bfbcb80-77c9-11e9-8b7f-24453f76c027.png)

**Exhibition**
![Exhibition_visit us](https://user-images.githubusercontent.com/1446139/57845116-3dc58f00-77c9-11e9-8767-78a14ae31ec5.png)

## To Do
- [ ] Decide what key info can be/should be exposed
- [ ] Decide whether to fit it inside the info box, or split it into two?
- [ ] Implement solution in Prismic

## Considerations
- Often the location of an event isn't confirmed until the last minute. The Events team make adjustments based on judgement of popularity of an event. 
- The Info Desk is often the meeting point for an event
- Hearing loops may be fixed, or portable loops may be available - these will be dependent on event location.

**Comment by jennpb** [comment]
Spoke to @dannybirchall about this. Work is currently underway as part of the Access audit (in building), and the content team will be working this into pair writing of events.

**Comment by jennpb** [comment]
@dannybirchall Should I keep this ticket open for the Experience team, or is it primarily a content issue?

Happy to discuss any changes to yellow info box, but if that's not currently under consideration, I will close this.
